syntax = "proto3";

package status;
option go_package = "github.com/DotAScripter/helm-demo-project/greetings/proto/status";
option java_package = "com.hdp.jpod.proto";

service Status {
  rpc CheckStatus (StatusCheckRequest) returns (StatusCheckResponse) {}
}

message StatusCheckRequest {
  string service_name = 1;
}

message StatusCheckResponse {
  enum ServiceStatus {
    UNKNOWN = 0;
    OK = 1;
    NOK = 2;
  }
  ServiceStatus status = 1;
}
